PowerEditor/src/ScintillaComponent/AutoCompletion.cpp

```diff
@@ -58,6 +58,8 @@ bool AutoCompletion::showApiComplete()
 		return false;
 
 	_pEditView->execute(SCI_AUTOCSETSEPARATOR, WPARAM(' '));
+	_pEditView->execute(SCI_AUTOCSETTYPESEPARATOR, WPARAM('?'));
+	_pEditView->execute(SCI_REGISTERIMAGE, 1000, LPARAM(xpmfn));

```

Comment:
It's better to use a named constant for image ID instead of a magic number.

Reply:
named constant?  Sorry my C++ skills are pretty low, do you mean like:

```
#define FUNC_IMG_ID 1000
```

Reply:
Yes, something like that will do.

Reply:
Try calling intToString() from "Common.h"

Reply:
Thanks @mere-human !  That worked, so I cleaned up a bit and pushed a new commit on this PR.

Cheers.